### ISSUE TITLE
rails5 minitest can't run because  undefined method `blank?' for class `Minitest::Mock'

### ISSUE THREAD

**Issue by tianlu1677** [blowmage/minitest-rails#197]
I use minitest-rails with master branch for rails5. I just touch `test/user_test.rb` and  `rake test`. 
I have this error 
```
DEPRECATION WARNING: alias_method_chain is deprecated. Please, use Module#prepend instead. From module, you can access the original method using super. (called from <top (required)> at /Users/jacksontom/Dropbox/daily/qdaily4_app/config/environment.rb:5)
rake aborted!
NameError: undefined method `blank?' for class `Minitest::Mock'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/minitest-5.9.0/lib/minitest/mock.rb:27:in `undef_method'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/minitest-5.9.0/lib/minitest/mock.rb:27:in `block in <class:Mock>'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/minitest-5.9.0/lib/minitest/mock.rb:26:in `each'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/minitest-5.9.0/lib/minitest/mock.rb:26:in `<class:Mock>'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/minitest-5.9.0/lib/minitest/mock.rb:10:in `<module:Minitest>'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/minitest-5.9.0/lib/minitest/mock.rb:3:in `<top (required)>'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/activesupport-5.0.0.rc1/lib/active_support/dependencies.rb:293:in `require'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/activesupport-5.0.0.rc1/lib/active_support/dependencies.rb:293:in `block in require'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/activesupport-5.0.0.rc1/lib/active_support/dependencies.rb:259:in `load_dependency'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/activesupport-5.0.0.rc1/lib/active_support/dependencies.rb:293:in `require'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/bundler/gems/minitest-rails-2a7b5ffaeb4f/lib/minitest/rails.rb:4:in `<top (required)>'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/activesupport-5.0.0.rc1/lib/active_support/dependencies.rb:293:in `require'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/activesupport-5.0.0.rc1/lib/active_support/dependencies.rb:293:in `block in require'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/activesupport-5.0.0.rc1/lib/active_support/dependencies.rb:259:in `load_dependency'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/activesupport-5.0.0.rc1/lib/active_support/dependencies.rb:293:in `require'
/Users/jacksontom/Dropbox/daily/qdaily4_app/test/test_helper.rb:4:in `<top (required)>'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/activesupport-5.0.0.rc1/lib/active_support/dependencies.rb:293:in `require'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/activesupport-5.0.0.rc1/lib/active_support/dependencies.rb:293:in `block in require'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/activesupport-5.0.0.rc1/lib/active_support/dependencies.rb:259:in `load_dependency'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/activesupport-5.0.0.rc1/lib/active_support/dependencies.rb:293:in `require'
/Users/jacksontom/Dropbox/daily/qdaily4_app/test/controllers/user_controller_test.rb:1:in `<top (required)>'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/activesupport-5.0.0.rc1/lib/active_support/dependencies.rb:293:in `require'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/activesupport-5.0.0.rc1/lib/active_support/dependencies.rb:293:in `block in require'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/activesupport-5.0.0.rc1/lib/active_support/dependencies.rb:259:in `load_dependency'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/activesupport-5.0.0.rc1/lib/active_support/dependencies.rb:293:in `require'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/railties-5.0.0.rc1/lib/rails/test_unit/test_requirer.rb:11:in `block in require_files'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/railties-5.0.0.rc1/lib/rails/test_unit/test_requirer.rb:10:in `each'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/railties-5.0.0.rc1/lib/rails/test_unit/test_requirer.rb:10:in `require_files'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/railties-5.0.0.rc1/lib/rails/test_unit/minitest_plugin.rb:76:in `plugin_rails_init'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/minitest-5.9.0/lib/minitest.rb:80:in `block in init_plugins'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/minitest-5.9.0/lib/minitest.rb:78:in `each'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/minitest-5.9.0/lib/minitest.rb:78:in `init_plugins'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/minitest-5.9.0/lib/minitest.rb:129:in `run'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/railties-5.0.0.rc1/lib/rails/test_unit/minitest_plugin.rb:64:in `rake_run'
/Users/jacksontom/.rvm/gems/ruby-2.3.0/gems/railties-5.0.0.rc1/lib/rails/test_unit/testing.rake:15:in `block in <top (required)>'
Tasks: TOP => test
(See full trace by running task with --trace)
```

gem 

```
gem 'minitest-rails', github: 'blowmage/minitest-rails', branch: 'master'
```

test/user_test.rb
```
require "test_helper"

class UserTest < ActiveSupport::TestCase
  def user
    @user ||= User.new
  end

  def test_valid
    assert user.valid?
  end
end

```

test_helper.rb

```
ENV["RAILS_ENV"] = "test"
require File.expand_path("../../config/environment", __FILE__)
require "rails/test_help"
require "minitest/rails"

# To add Capybara feature tests add `gem "minitest-rails-capybara"`
# to the test group in the Gemfile and uncomment the following:
# require "minitest/rails/capybara"

# Uncomment for awesome colorful output
# require "minitest/pride"

class ActiveSupport::TestCase
  # Setup all fixtures in test/fixtures/*.yml for all tests in alphabetical order.
  fixtures :all
  # Add more helper methods to be used by all tests here...
end

```

**Comment by sivagollapalli** [comment]
@blowmage Just checked. No issue with stable version `5.0.0`. I think we close this issue.

**Comment by blowmage** [comment]
Great. Thanks!

**Comment by bmatzelle** [comment]
FYI, for anyone seeing this issue.  It's not a MiniTest::Mock bug.  It's actually a big with Ruby 2.2/2.3/2.4, see issue [#13096][1], which was [fixed in Ruby 2.2.7][2].  I updated my 2.2.3 version of Ruby to 2.2.7 and it works fine now.  

[1]: https://bugs.ruby-lang.org/issues/13096
[2]: https://svn.ruby-lang.org/repos/ruby/tags/v2_2_7/ChangeLog
